### ISSUE TITLE
Ensure private file system is setup during install.

### ISSUE THREAD

**Issue by giteshk** [apigee/apigee-devportal-kickstart-drupal#474]
It has been observed that the Apigee Edge module does not work without private file system being setup.

1. Add a requirement in the install step to ensure private file system is setup.
2. Ensure the wizard stores edge connection directly to the private file system. 
3. Remove warnings displayed, during installation, to move credentials to private file system instead of configuration store

**Comment by phdhiren** [comment]
Duplicate of #468

Please note that setting up private file system in settings.php is manual step.

> In Drupal 8, you can no longer set private file system via UI. You set it in your settings.php file.

https://www.drupal.org/docs/8/core/modules/file/overview#private-file-system


Added private file system as requirement in pull #472 - Once private file system is in place other points will be taken care automatically.

**Comment by giteshk** [comment]
Can you confirm if items 2 and 3 are addressed?

**Comment by phdhiren** [comment]
Yes, Once private file system is in place, OAuth / Apigee X connection settings will be stored in private file system and will not show a warning to move credentials to private file system. 
As basic connection do not require private file system, the requirement of private file system in not mandatory but highly recommended in pull #472
